This reverts the dependency to wayland-protocols to version 1.17, the version currently available.
--- a/protocol/meson.build
+++ b/protocol/meson.build
@@ -1,7 +1,7 @@
 dep_scanner = dependency('wayland-scanner', native: true)
 prog_scanner = find_program(dep_scanner.get_pkgconfig_variable('wayland_scanner'))
 
-dep_wp = dependency('wayland-protocols', version: '>= 1.18')
+dep_wp = dependency('wayland-protocols', version: '>= 1.17')
 dir_wp_base = dep_wp.get_pkgconfig_variable('pkgdatadir')
 
 install_data(
